src/probnum/filtsmooth/gaussfiltsmooth/extendedkalman.py

```diff
@@ -136,6 +136,7 @@ def __init__(
         mde_atol=1e-5,
         mde_rtol=1e-5,
         mde_solver="LSODA",
+        forward_implementation="classic",
```

Comment:
thanks for doing this! Is this argument in the docstring?

Reply:
No, it also is not for `LinearSDE` and `DiscreteLinearGaussian` (see also 3. in https://github.com/probabilistic-numerics/probnum/pull/374#issue-610816053).

Reply:
Hm okay, I can live with that :)

Reply:
I can introduce the argument in the docstring if you'd feel more comfortable with that :)